aur.in

```diff
@@ -22,7 +22,9 @@ if [[ -z $1 ]]; then
     exit 1
 fi
 
-readonly PATH=$lib_dir:$PATH
+if [[ "$PATH" != "$lib_dir:"* ]]; then
```

Comment:
nice idea, should we maybe search _anywhere_ in PATH, not only in the beginning, for completeness?

Reply:
`$lib_dir` needs to be in the first spot to guarantee that the invocations of official commands use the official scripts.

Otherwise it would allow me to do sneaky things like: `PATH=~/bin:/usr/lib/aurutils/` and that would allow me to have overrides for official scripts in `~/bin`. Breaking any expectations that 3rd parties might have on what `aur x` does.

Reply:
I've been tempted to ask for the ability to override the default `$lib_dir` because it makes debugging slightly easier. (making `aur` run directly from my clone). 

But that has the same issues as mentioned above, i.e. can be used as a way to override  default behaviour which I don't want to encourage.

Does this makes sense? Are we being overly protective?

Reply:
FWIW, in `1.5.3` anyone could easily override scripts by placing a local copy higher up in PATH, and it didn't cause any real issues that I can remember.

Reply:
In that's desired, they can void aurutils warranty and place a modified aur wrapper higher in the `PATH` with the modified $`PATH` check/`lib_dir`... and assume the responsibility for any unexpected behavior.

Reply:
I think #535 offers a nice alternative